src/linux/resources/jd-gui.desktop

```diff
@@ -2,6 +2,7 @@
 Comment=Java Decompiler JD-GUI
 Terminal=false
 Name=JD-GUI
-Exec=java -jar /opt/jd-gui/jd-gui.jar
+Exec=java -jar /opt/jd-gui/jd-gui.jar %F
 Type=Application
 Icon=jd-gui
+MimeType=application/java-archive;application/java-vm
```

Comment:
I take my former comment back:  I misunderstood that Mimetype was supported files the application could open according to the [desktop entry spec](http://standards.freedesktop.org/desktop-entry-spec/latest/ar01s05.html).